EventSource/EventSource.swift

```diff
@@ -47,15 +47,12 @@ open class EventSource: NSObject, URLSessionDataDelegate {
         self.receivedString = nil
         self.receivedDataBuffer = NSMutableData()
 
-
-        var port = ""
-        if let optionalPort = self.url.port {
-            port = String(optionalPort)
-        }
+        let port = String(self.url.port ?? 80)
```

Comment:
Shouldn't it be port 8080 instead of just 80?

Reply:
I couldn't find a standard port for event source but as it's just an http request I think the right port would be 80 if not specified.

Reply:
Got it!